lib/absinthe/phase/document/validation/no_fragment_cycles.ex

```diff
@@ -45,7 +45,8 @@ defmodule Absinthe.Phase.Document.Validation.NoFragmentCycles do
           graph
           |> :digraph_utils.topsort()
           |> Enum.reverse()
-          |> Enum.map(&Map.fetch!(fragments, &1))
+          |> Enum.map(&Map.get(fragments, &1))
+          |> Enum.reject(&is_nil/1)
```

Comment:
So this was the simplest thing I could think of. Not sure, if this is the right approach?

Also, we can do this in one pass, if necessary.